index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Animated Toolbar icons</title>
+    <script src="https://kit.fontawesome.com/e88d33689c.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" type="text/css"
+        href="https://cdn.jsdelivr.net/gh/loadingio/loading.css@v2.0.0/dist/loading.min.css" />
+    <link href="./style.css" rel="stylesheet" />
+</head>
+
+<body>
+    <header>
+        <div class="wrapper">
+            <nav>
+                <ul>
+                    <li class="calendar">
+                        <a href="#"><img src="./calendar-icon.gif"></a>
+                    </li>
+                    <li class="search">
+                        <a href="#"><img src="./search-icon.gif"></a>
```

Comment:
I noticed you had the SVG version of these icons. You can use them directly instead of the gift.

You could take a look at `animation` and `transform: scale()` properties to learn how to animate an element and doing that "pulse" effect.